### PR TITLE
[ADD] helpdesk_type

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Review by nelsonramirezs — CHANGES_REQUESTED** [review]
Manifest Name: Helpdesk Motive" @max3903

**Review by nelsonramirezs — APPROVED** [review]

**Review by nelsonramirezs — APPROVED** [review]

File: helpdesk_type/views/helpdesk_ticket_type.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    <record id="action_helpdesk_ticket_type" model="ir.actions.act_window">
+        <field name="name">Helpdesk Ticket Type</field>
+        <field name="res_model">helpdesk.ticket.type</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <menuitem
+        name="Types"
```
Comment (bodedra) [inline helpdesk_type/views/helpdesk_ticket_type.xml]:
```suggestion
        name="Ticket Types"
```

**Review by bodedra — APPROVED** [review]
LGTM. Added menu name suggestion.

**Pull request merged**